src/recipe/recipe.controller.ts

```diff
@@ -41,11 +44,24 @@ export class RecipeController {
     const fetchedRecipe = await this.recipeService.fetchRecipe(id);
 
     return FetchRecipeResponse.from(fetchedRecipe);
-      
+  }
+
   @Get()
+  @UseGuards(JwtAuthGuard)
   async fetchRecipes(): Promise<FetchRecipesResponse> {
     const fetchedRecipes = await this.recipeService.fetchAllRecipes();
 
     return FetchRecipesResponse.from(fetchedRecipes);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @Patch(':id')
```

Comment:
quick question: how does the update work right now? Is it updating (replacing) the whole object or is it updating only the fields we provide in the request?

- If the first approach then this should be a `@Put`
- If the second approach then `@Patch` is fine 👍🏻 (I believe it's this one in this case)